test/integration/com/unifina/com/unifina/domain/marketplace/PaidSubscriptionIntegrationSpec.groovy

```diff
@@ -17,7 +17,7 @@ class PaidSubscriptionIntegrationSpec extends IntegrationSpec {
 			username: "subscription-service-integration-spec-1@streamr.com",
 			password: "xxx",
 			name: "Subscription Service Integration Spec 1",
-			timezone: "UTC"
+			lastLogin: new Date(0),
```

Comment:
Can remove `lastLogin` if you set the default field value in `SecUser.groovy` (also elsewhere in this file)